test/e2e/fpgaadmissionwebhook/fpgaadmissionwebhook.go

```diff
@@ -41,22 +41,31 @@ func describe() {
 	f := framework.NewDefaultFramework("webhook")
 
 	ginkgo.It("mutates created pods to reference resolved AFs", func() {
-		checkPodMutation(f, "fpga.intel.com/d5005-nlb3-preprogrammed",
+		checkPodMutation(f, f.Namespace.Name, "fpga.intel.com/d5005-nlb3-preprogrammed",
 			"fpga.intel.com/af-bfa.f7d.v6xNhR7oVv6MlYZc4buqLfffQFy9es9yIvFEsLk6zRg")
 	})
 
 	ginkgo.It("mutates created pods to reference resolved Regions", func() {
-		checkPodMutation(f, "fpga.intel.com/arria10.dcp1.0-nlb0-orchestrated",
+		checkPodMutation(f, f.Namespace.Name, "fpga.intel.com/arria10.dcp1.0-nlb0-orchestrated",
 			"fpga.intel.com/region-ce48969398f05f33946d560708be108a")
 	})
 
 	ginkgo.It("mutates created pods to reference resolved Regions in regiondevel mode", func() {
-		checkPodMutation(f, "fpga.intel.com/arria10.dcp1.0",
+		checkPodMutation(f, f.Namespace.Name, "fpga.intel.com/arria10.dcp1.0",
 			"fpga.intel.com/region-ce48969398f05f33946d560708be108a")
 	})
+
+	ginkgo.It("doesn't mutate a pod if it's created in a namespace different from mappings'", func() {
```

Comment:
closes #406 too?

Reply:
Right. Added the closing clause to the description.